setup.py

```diff
@@ -28,7 +28,8 @@
 # http://pypi.python.org/pypi/setuptools
 
 EXTRAS = {
-    'adal': ['adal>=1.0.2']
+    'adal': ['adal>=1.0.2'],
+    'google-auth': ['google-auth>=1.0.0']
```

Comment:
Shall we specify `google-auth>=1.0.1`, to be consistent with the requirements.txt removal?

Reply:
> Shall we specify `google-auth>=1.0.1`, to be consistent with the requirements.txt removal?

Yeah, That's fine we can make it consistent to version 1.0.1

Reply:
That change has been done